js/lib/Country.js

```diff
@@ -81,22 +81,22 @@ var Country = function (options) {
 
 	this.locale = this.locale || new Locale();
 	localeinfo = new LocaleInfo(this.locale);
```

Comment:
Need to load this asynchronously in case Country itself is loaded asynchronously:

```
new LocaleInfo(this.locale, {
  loadParams: loadParams,
  sync: sync,
  onLoad: function(li) {
    this.locale = li;
etc.
```